enhancements/network/routed-ingress-support-for-primary-udn-attached-vms-with-static-ips.md

```diff
@@ -0,0 +1,789 @@
+  user (e.g. the desired IP allocation is already in use).
+- the `IPAMClaim` CRD spec will need to be updated, adding `MAC` and `IPs`
+  requests.
+- OVN-Kubernetes will need to try to apply the MAC and IP addresses requested
+  in its spec. If any conflicts are found for either MAC or IPs, the error will
+  be persisted in the `IPAMClaim` status (in the conditions array).
+- OVN-Kubernetes will record the requested IPs in the status sub-resource
+  (already does so today), along with a `Success` condition.
+
+This option has the network admin create the IPAMClaim, and request a MAC and
+IP addresses for the VM; in a second stage of the implementation, MTV (or any
+other component able to introspect the source cluster for the VM's MAC and IPs)
+will create the IPAMClaim on behalf of the admin user.
+
+###### Admin user flows for de-centralized approach
+
+The admin user will have to perform the following operations:
+- provision the cluster UDN CR having the overlay definition.
+  - layer2 topology ; IPAM lifecycle `Persistent` are a **must**
+- provision the `IPAMClaim` object, whose name **must** be created like
+  `<vm name>.<logical network name>`. The `IPAMClaim` must have both the IP and
+  MAC addresses for the VM attachment. This is only required until another tool
+  automatically provisions the `IPAMClaim`s on behalf of the admin
+
+As indicated above, when a tool exists that can introspect the source cluster
+to learn the VM's IP addresses, the admin user will only require provisioning
+the cluster UDN CR.
+
+###### VM owner flows for de-centralized approach
+
+The VM owner just has to use MTV to import the VM into CNV.
+
+## Test Plan
+
+* E2E upstream and downstream jobs covering VM creation with requested
+  IP/MAC/gateway configuration for both IPv4 and dual-stack configurations
+* E2E downstream jobs covering a VM (having a network configuration which must
+  be kept) import via MTV is successful for both IPv4 and dual-stack
+  configurations
+* E2E tests covering the existing features (network policies, IP spoof
+  protection, services) for an imported VM works as expected for both IPv4 and
+  dual-stack configurations
+
+## Graduation Criteria
+
+### Dev Preview -> Tech Preview
+
+There will be no Dev Preview for this feature.
+
+### Tech Preview -> GA
+
+Targeting GA in OCP version 4.20.
+
+### Removing a deprecated feature
+
+The annotation currently being used for indicating to OVN-Kubernetes what is
+the `IPAMClaim` to use for the primary UDN attachment will be deprecated, and
+we plan on using the [multus default annotation](#cnv-ovnk-api) for it.
+
+The deprecation strategy is described in the OVN-Kubernetes
+[OKEP](https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5238).
+
+## Upgrade / Downgrade Strategy
+
+N/A
+
+## Version Skew Strategy
+
+N/A
+
+## Operational Aspects of API Extensions
+
+The proposed `IPPool` CRD must be provisioned by the admin (or the source
```

Comment:
I'm a bit confused. I thought this centralized approach was not the primary chosen solution.
But it appears here as if it is.